conf/evolutions/default/6.sql

```diff
@@ -0,0 +1,33 @@
+# --- Support for auto tuning spark
+# --- !Ups
+
+ALTER TABLE tuning_algorithm ADD UNIQUE KEY (optimization_algo, optimization_algo_version);
+ALTER TABLE tuning_job_execution ADD COLUMN is_param_set_best tinyint(4) default 0;
+ALTER TABLE tuning_job_definition ADD COLUMN tuning_disabled_reason text;
+
+INSERT INTO tuning_algorithm VALUES (2, 'SPARK', 'PSO', '2', 'RESOURCE', current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (10,'spark.executor.memory',2, 2048, 1024, 10240, 1024, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (11,'spark.memory.fraction',2, 0.6, 0.1, 0.9, 0.1, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (12,'spark.memory.storageFraction', 2, 0.5, 0.1, 0.9, 0.1, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (13,'spark.executor.cores', 2, 1 , 1, 1, 1, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (14,'spark.yarn.executor.memoryOverhead', 2, 384, 384, 1024, 100, 0, current_timestamp(0), current_timestamp(0));
```

Comment:
Can you add a comment on the background behind these constants?

Reply:
This was actually not needed so I removed it altogether.